src/libs/WorkflowUtils.ts

```diff
@@ -117,9 +117,23 @@ function convertPolicyEmployeesToApprovalWorkflows({employees, defaultApprover,
             return -1;
         }
 
+        if (b.isDefault) {
+            return 1;
+        }
+
         return (a.approvers[0]?.displayName ?? '-1').localeCompare(b.approvers[0]?.displayName ?? '-1');
     });
 
+    // Add a default workflow if one doesn't exist (no employees submit to the default approver)
```

Comment:
As I understand, It should be: 
```suggestion
    // Add a default workflow if one doesn't exist (every employees submit to the default approver)
```

Reply:
'no employees submit to the default approver' is actually correct, this is an egde case where nobody submits to the default approver and it wasn't added to the workflow list